coda-prototype/package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "description": "Coda is an interactive tool that helps you label short text datasets!",
   "main": "ui.js",
-  "dependencies": {},
+  "dependencies": {
+    "d3": "^4.11.0"
```

Comment:
I think this will need an adjustment to the licence file? https://github.com/lukechurch/coda/blob/master/coda-prototype/LICENSE-3RD-PARTY.txt